.github/workflows/pr-testing-with-test-project.yml

```diff
@@ -14,8 +14,28 @@ on:
       - '!docs/**'
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      should_test: ${{ steps.filter.outputs.changes }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v3
```

Comment:
just one thing, for security reasons we always try to remember to not use specific release but commitId

can you switch to something like in the example please: https://github.com/asyncapi/generator/blob/master/.github/workflows/bump.yml#L19